packages/vite-plugin-svelte/src/utils/dependencies.ts

```diff
@@ -174,6 +162,19 @@ function is_common_without_svelte_field(dependency: string): boolean {
 	);
 }
 
+export function needsOptimization(dep: string, localRequire: NodeRequire): boolean {
+	const depData = resolveDependencyData(dep, localRequire);
+	if (!depData) return false;
+	const pkg = depData.pkg;
+	// only optimize if is cjs, using the below as heuristic
```

Comment:
add a link to #162 so readers have a better chance of figuring out why we do this?